book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author, rentals = [])
+    @title = title
+    @author = author
+    @rentals = rentals
+  end
```

Comment:
- [ ] **[OPTIONAL]** You probably don't need to define the empty **`rentals`** in the parameters like this to accomplish the initialization. The following code snippet works too. This way there will be a lesser number of arguments when instantiating a Book object.
  
  ```
  def initialize(title, author)
      @title = title
      @author = author
      @rentals = []
    end
  ```